controllers/healthcheck_controller.go

```diff
@@ -869,7 +896,7 @@ func (r *HealthCheckReconciler) parseWorkflowFromHealthcheck(log logr.Logger, hc
 	if activeDeadlineSeconds := data["spec"].(map[string]interface{})["activeDeadlineSeconds"]; activeDeadlineSeconds == nil {
 		data["spec"].(map[string]interface{})["activeDeadlineSeconds"] = &timeout
 	}
-	log.Info("HealthCheck with Workflow", "Spec:", data)
+	//log.Info("HealthCheck with Workflow", "Spec:", data)
```

Comment:
Minor. This should be a Debug level log or removed.

Reply:
Thanks.. Done.